tests/components/flexit_bacnet/test_config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Test the Flexit Nordic (BACnet) config flow."""
+import asyncio.exceptions
+
+from flexit_bacnet import DecodingError
+import pytest
+
+from homeassistant.components.flexit_bacnet.const import DOMAIN
+from homeassistant.const import CONF_DEVICE_ID, CONF_IP_ADDRESS
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from ...common import MockConfigEntry
+from .conftest import _patch_update
```

Comment:
I don't think mypy allows you to import privates like this, maybe move the config flow fixtures back over (I know I told to move them there in the first place)

Reply:
Okay, will do!